desktop/coworker/server.py

```
@@ -1740,6 +1741,57 @@ def people_meetings_reject(person_id: str, evidence_id: str):
+            return JSONResponse({"error": "not found"}, status_code=404)
+        if app.state.drive is None:
+            return JSONResponse({"error": "Drive is not connected"}, status_code=400)
+        body = await request.json()
+        query = str(body.get("query") or "").strip()
+        if not query:
+            return JSONResponse({"error": "query is required"}, status_code=400)
+        try:
+            result = app.state.drive.search(
+                query, max_results=int(body.get("max_results") or 10)
+            )
+        except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)